client/src/components/ElementActions/SaveAction.js

```diff
@@ -34,6 +40,14 @@ const SaveAction = (MenuComponent) => (props) => {
       { type: type.title }
     );
 
+    // https://redux-form.com/8.3.0/examples/remotesubmit/
+    submitForm();
```

Comment:
There was a callback below this to show toast notifications, however I don't see us getting this to work because redux-forms 'submit()' function used for triggering a remote submit isn't thenable nor does it allow for adding a callback or have any other method that allows us to do stuff 'onAfterSuccess / onAfterFailure'